scripts/release/common/setup.sh

```diff
@@ -105,6 +105,7 @@ fi
 
 sudo usermod -a -G docker ubuntu
 sg docker "docker pull centos:7"
+sg docker "docker pull centos:8"
```

Comment:
Might want to replace this with the stream image, but not really a blocker.

Reply:
Updated to pull quay.io/centos/centos:stream8.